e2e/tests/api-driven/src/globalHelpers.ts

```diff
@@ -1,5 +1,9 @@
-import { TEST_EMAIL } from "../../ui-driven/src/helpers/globalHelpers";
-import { $admin } from "./client";
+import { $admin } from "./client.js";
+
+// Gov.uk Notify requests testing service use smoke test email addresses
+// see https://docs.notifications.service.gov.uk/rest-api.html#smoke-testing
+export const TEST_EMAIL =
```

Comment:
Previously this was being imported from the Playwright tests - I was hitting issues as I converted these one at a time to ESM.

Repetition here seemed better even when both projects were converted as you might not expect a change in UI tests to break API tests.